openlibrary/plugins/upstream/account.py

```diff
@@ -325,7 +325,9 @@ def POST(self):
         if error:
             return self.render_error(error, i)
 
-        expires = (i.remember and 3600 * 24 * 7) or ""
+        expires = i.remember and 3600 * 24 * 7
```

Comment:
```suggestion
        expires = 3600 * 24 * 7 if i.remember else ""
```

This doesn't look right. Looking at web.py 's code, it creates a dict with all the pieces of the cookie, and then stringifies them with this:

```py
        for key, value in items:
            if value == "":
                continue
            if key not in attrs:
                continue
            if key == "expires" and isinstance(value, int):
                append("%s=%s" % (self._reserved[key], _getdate(value)))
            elif key == "max-age" and isinstance(value, int):
                append("%s=%d" % (self._reserved[key], value))
            elif key == "comment" and isinstance(value, str):
                append("%s=%s" % (self._reserved[key], _quote(value)))
            elif key in self._flags:
                if value:
                    append(str(self._reserved[key]))
            else:
                append("%s=%s" % (self._reserved[key], value))
```


i.e. it has a special clause for `""`. My guess would be setting this to False might cause cookie issues, cause it'll stringify `False`; i.e. `expires=False`. That doesn't seem valid.